lib/challenge.js

```diff
@@ -274,7 +354,11 @@ let nestedArrays = [
 ];
 
 function prompt13() {
-  // YOUR CODE HERE
+  // for (let i = 0; i < nestedArrays.length; i++) {
+  //   for (let j = 0; j < 3; j++) {
+  //     console.log(nestedArrays[i][j]);
+  //   }
+  // }
 }
```

Comment:
Nice!  Is there away to make the inner loop dynamic rather than hard coding the length to `3`? (resubmission not required)